<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="btnDialogErsterModal">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>dialogErster modal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDialogErsterNonModal">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>dialogErster non modal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getTextFromDialogErster">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>getText</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="edtLog">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>241</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShowWidget2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>show Widget2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHideWidget2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>hide Widget2</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
